tests/layer_tests/tensorflow_tests/test_tf_tensor_scatter_update.py

```diff
@@ -0,0 +1,78 @@
+# Copyright (C) 2018-2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+import platform
+import pytest
+import tensorflow as tf
+from common.tf_layer_test_class import CommonTFLayerTest
+
+rng = np.random.default_rng(872173)
+
+
+class TestTensorScatterUpdate(CommonTFLayerTest):
+    def _prepare_input(self, inputs_info):
+        assert 'tensor:0' in inputs_info
+        assert 'indices:0' in inputs_info
+        assert 'updates:0' in inputs_info
+
+        tensor_shape = inputs_info['tensor:0']
+        updates_shape = inputs_info['updates:0']
+        indices_shape = inputs_info['indices:0']
+
+        inputs_data = {}
+        if np.issubdtype(self.data_type, np.floating):
+            inputs_data['tensor:0'] = rng.uniform(-5.0, 5.0, tensor_shape).astype(self.data_type)
+            inputs_data['updates:0'] = rng.uniform(-5.0, 5.0, updates_shape).astype(self.data_type)
+        elif np.issubdtype(self.data_type, np.signedinteger):
+            inputs_data['tensor:0'] = rng.integers(-8, 8, tensor_shape).astype(self.data_type)
+            inputs_data['updates:0'] = rng.integers(-8, 8, updates_shape).astype(self.data_type)
+        else:
+            inputs_data['tensor:0'] = rng.integers(0, 8, tensor_shape).astype(self.data_type)
+            inputs_data['updates:0'] = rng.integers(0, 8, updates_shape).astype(self.data_type)
+        
+        inputs_data['indices:0'] = rng.integers(0, min(tensor_shape), indices_shape).astype(self.indices_type)
```

Comment:
it can be a case that some indices can be repeating that is not good and can give sporadic issues. So I propose to generate a set of all possible indices tuples for the given case and select index tuples from this set using rng.choise without repeat.